tests/app/main/views/test_buyers.py

```diff
@@ -152,18 +152,46 @@ def test_post_page_should_only_be_accessible_to_specific_user_roles(self, role,
         actual_code = response.status_code
         assert actual_code == expected_code, "Unexpected response {} for role {}".format(actual_code, role)
 
-    def test_admin_user_can_add_a_new_buyer_domain(self):
+    @pytest.mark.parametrize("new_buyer_domain", (
+        "kev.uk",
+        "zz",
+        "keyes.ORG.uk",
+        ".GOV",
+        "om",  # a suffix of "com", but not if we're (correctly) considering an implicit preceding separator
```

Comment:
This is an interesting one. I wonder if we should also have a minimum length of say, 3, set? Is there any realistic situation where a public sector buyer is going to have a 2 character email domain? Out of scope of this PR, just thinking out loud...

Reply:
I guess this would go in the API's schema validation though

Reply:
`.GOV` will fail API validation - prob best not to use it as an example given we're mocking the response

Reply:
I think actually the API's json schema based validation will stop anything < 2 chars.

Reply:
Yup, as discussed what's actually going on here is I'm reying to squeeze unit-y tests of the validator into the _view_'s unit tests. Which is what makes it inappropriate to put otherwise-failing examples in here. Will split the parametrization out into separate tests.